db/db.js

```diff
@@ -11,12 +11,37 @@ var database = mongoose.connection;
 var orderSchema = new Schema({
   customerName:{
     type: String,
+    var testName=validator.isAlphanumeric(str [, locale]);
+    if (testname===false){
+      message:'{Value} is not a valid name, please enter again'
```

Comment:
this is not the correct string building code in javascript, see here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals for reference. also testname should be testName.